src/components/Person/Person.jsx

```diff
@@ -1 +1,30 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  let contentPartner;
+  let contentAge;
+
+  if ('age' in person) {
```

Comment:
Nope, definitely not like that. You never want to assign an html-like code to a variable inside the react component in that manner. 

If you wish to extract the logic with age and partner - there should be a helper function for it, like:
```

const renderAge = (age) => {
  if(person?.age) {
    return <p className="Person__age">I am {person.age}</p>;
  }
  return null;
}

...
return (
    <section className="Person">
      <h2 className="Person__name">My name is {person.name}</h2>
      {renderAge()}
    </section>
)
```

However, this approach may also be simplified in this case by just: 

```
  return (
    <section className="Person">
      <h2 className="Person__name">My name is {person.name}</h2>
      {person.age && <p className="Person__age">I am {person.age}</p>}
    </section>
  );
```